### DISCUSSION TITLE
Is possible send empty array by FormData?

### DISCUSSION THREAD

**Discussion by wildfrontend** [axios/axios#5577] [Q&A] [answered]
i use Automatic serialization to FormData to send empty array

but it look like ignore  empty value

is automatic object serialization to a FormData object can send empty array?

**Comment by DigitalBrainJS** [comment, marked as answer]
An empty array [can't be represented in FormData](https://stackoverflow.com/questions/32913127/how-to-submit-empty-array-from-html-form-post-to-php) as this is a flat data structure that only emulates data nesting, but you can send JSON string as a field value.
```js
await axios.postForm("http://localhost/", {
  emptyArr: JSON.stringify([])
});
```
Unfortunately, at the moment server frameworks (like `multer`) do not support automatic deserialization as there is no established practice of serializing JSON into FormData fields, so you need to do it manually
